index.html

```diff
@@ -23,6 +23,6 @@
         <img src="img/js-badge.svg" alt="11" id = "img10" class = "card">
         <img src="img/js-badge.svg" alt="12" id = "img11" class = "card">
       </main>
-        <script src="main.js"></script>
+        <script src="src/main.js"></script>

```

Comment:
You don't have to move everything to `src` folder to use `eslint` but in general it looks better :)